frontend/www/js/omegaup/components/homepage/CoderOfTheMonth.vue

```diff
@@ -51,7 +51,12 @@
       </template>
     </div>
     <div class="card-footer">
-      <a href="/coderofthemonth/">{{ T.coderOfTheMonthFullList }}</a>
+      <a
+        v-bind:href="
+          category == 'female' ? '/coderofthemonth/female' : /coderofthemonth/
```

Comment:
```suggestion
          category == 'female' ? '/coderofthemonth/female/' : '/coderofthemonth/'
```

sin los `''`, lo segundo es una [expresión regular](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions), no una cadena!